temporal_capi/Cargo.toml

```diff
@@ -12,6 +12,7 @@ include = [
     "src/**/*",
     "tests/**/*",
     "Cargo.toml",
+    "LICENSE*",
```

Comment:
this doesn't do anything, LICENSE doesn't exist.

Reply:
I explicitly listed the license filenames then. I'm assuming that should then include them